pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
I couldn't find any other way to make it build in Eclipse.  If you're using Eclipse, how do you get rid of the error without changing this config?